test/init/__snapshots__/init.test.js.snap.webpack4

```diff
@@ -457,7 +512,59 @@ module.exports = {
                 test: /\\\\.s[ac]ss$/i,
                 use: ['style-loader', 'css-loader', 'sass-loader'],
             },
+            {
+                test: /\\\\.(eot|svg|ttf|woff|woff2|png|jpg|gif)$/,
```

Comment:
Let's use `i` flag for regexp, but let's do it in the next PR

Reply:
Okay, on it!